query-engine/driver-adapters/js/driver-adapter-utils/package.json

```
@@ -1,6 +1,6 @@
 {
-  "name": "@jkomyno/prisma-driver-adapter-utils",
-  "version": "0.3.0",
+  "name": "@prisma/driver-adapter-utils",
+  "version": "0.0.0",
```

Stay with current version (manually increased) until we have CI setup:
```suggestion
  "version": "0.3.0",
```